dbusmock/templates/bluez5.py

```diff
@@ -16,7 +16,11 @@
 # of the license.
 
 __author__ = 'Philip Withnall'
+__email__ = 'philip.withnall@collabora.co.uk'
```

Comment:
Is this required?

Reply:
Agreed,  please drop this. See commit 801e10abb8